.github/workflows/create-release.yml

```diff
@@ -1,11 +1,227 @@
 name: Create Release
+on:
+  workflow_dispatch:
+    inputs:
+      changelog_file:
```

Comment:
do we need this arg? this was prob copy pasta from the relying on reusable workflow.